proto/sentry_protos/snuba/v1/request_common.proto

```diff
@@ -0,0 +1,32 @@
+syntax = "proto3";
+
+package sentry_protos.snuba.v1;
+
+import "google/protobuf/timestamp.proto";
+import "sentry_protos/snuba/v1/trace_item_filter.proto";
+
+message RequestMeta {
+  uint64 organization_id = 1;
+  string cogs_category = 2;
+  string referrer = 3;
+  // can be empty
+  repeated uint64 project_ids = 4;
+  google.protobuf.Timestamp start_timestamp = 5;
+  google.protobuf.Timestamp end_timestamp = 6;
+  TraceItemName trace_item_name = 7;
```

Comment:
should probably be TraceItemType

Reply:
Why is that? I imagine a `type` to signify a different category which this does not. For example, issues and spans are not different types, they are both trace items, they are accessed with the same RPC calls, but they have different names.